packages/block-library/src/index.native.js

```diff
@@ -144,6 +144,7 @@ export const registerCoreBlocks = () => {
 		mediaText,
 		// eslint-disable-next-line no-undef
 		!! __DEV__ ? group : null,
+		spacer,
```

Comment:
Should it be able only for dev currently as group?

Reply:
Yes 👍 We'll keep it that way until we finish Slider improvements